Dockerfile

```diff
@@ -1,17 +1,52 @@
 # Copyright (c) 2019 Zededa, Inc.
 # SPDX-License-Identifier: Apache-2.0
-FROM debian:9.11
-RUN apt update && \ 
-    apt install -y g++ make libprotobuf-dev \
-                   libprotoc-dev protobuf-compiler \
-                   cmake libssl-dev libcurl4-openssl-dev uuid-dev
+FROM ubuntu:20.04 as build
+ARG IOTD_COMMIT_ID=15f59c8bd33b1fd8581a74ae6e5ea145c8cb1b9b
+ARG DEBIAN_FRONTEND=noninteractive
```

Comment:
Where is this arg used?

Reply:
on line 38

Reply:
OK, so this becomes a setenv which is then passed to apt-get etc. It would be good to add a comment about that.